microsoft/PowerToys
# [FancyZones] Overlapping zones selection algorithm - settings

## Summary of the Pull Request

**What is this about:**

In a previous PR #9447, we added support for different algorithms which should determine which zone gets selected when a user drags a window over a point contained inside several zones. The default is "smallest zone wins", and this is also respected in this PR. This PR exposes the option in PowerToys settings (only the new settings, for now).

**How does someone test / validate:** 

In order to test this, create a canvas zone layout with overlapping zones. For examples of such layouts, see #1167.
Change the setting and check whether the highlighted zone is the expected one.
## Quality Checklist

- [x] **Linked issue:** #1167
- [x] **Tests: See below**

Some unit tests fail because the logger isn't initialized.

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
